packages/dds/tree/src/feature-libraries/flex-tree/lazyField.ts

```diff
@@ -362,20 +362,6 @@ export class LazyValueField extends ReadonlyLazyValueField implements FlexTreeRe
 	}
 }
 
-export class LazyIdentifierField
```

Comment:
When fixing a stable assert message with referenced this type, I confirmed it was unused, and removed it.